DistroLauncher/WinTuiStrategy.cpp

```diff
@@ -0,0 +1,29 @@
+#include "stdafx.h"
+#include "ApplicationStrategyCommon.h"
+#include "WinTuiStrategy.h"
+
+namespace Oobe
+{
+    HRESULT WinTuiStrategy::do_autoinstall(const std::filesystem::path& autoinstall_file)
+    {
+        return internal::do_autoinstall(installer, autoinstall_file);
+    }
+
+    HRESULT WinTuiStrategy::do_install(Mode uiMode)
+    {
+        if (uiMode == Mode::Gui) {
+            wprintf(L"GUI mode is not supported on this platform.\n");
```

Comment:
(Non blocking)
I know this uses `wprintf` for consistency reasons, but do we want to change this in the future? Just some ideasd.
1. Sending to stderr
2. Useing iostream
3. Implementing or importing an existing logger at some point

Reply:
I've been dreaming with a logging facility. When the time comes, it will essentially be ` s/wprintf(\(.+\));/LOG(\1);/g
` .